pkg/translator/signalfx/from_metrics.go

```diff
@@ -146,10 +158,25 @@ func convertHistogram(in pmetric.HistogramDataPointSlice, name string, mt *sfxpb
 		count := int64(histDP.Count())
 		countDP.Value.IntValue = &count
 
-		sumName := name + "_sum"
-		sumDP := dps.appendPoint(sumName, mt, ts, dims)
-		sum := histDP.Sum()
-		sumDP.Value.DoubleValue = &sum
+		if histDP.HasSum() {
+			sumDP := dps.appendPoint(name+"_sum", mt, ts, dims)
+			sum := histDP.Sum()
+			sumDP.Value.DoubleValue = &sum
+		}
+
+		if histDP.HasMin() {
+			// Min is always a gauge.
+			minDP := dps.appendPoint(name+"_min", &sfxMetricTypeGauge, ts, dims)
```

Comment:
Why `sfxMetricTypeGauge` not `mt` for min and max?

Reply:
Because they are a gauge correct?

Reply:
Yes, they must be gauges all the time as we discussed